_config.yml

```diff
@@ -30,9 +30,8 @@ author: adasilva, achuinar
 
 # This URL is the main address for absolute links. Don't include a slash at the end.
 #
-#url: 'https://physicsmatters.physics.mcgill.ca'
```

Comment:
This file should not have been edited (it is being reverted to a previous version here.)